### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by CangShui** [KaylinOwO/Project-Apparatus#58]
Need Spawn Shotgun function

**Comment by JadeVain** [comment]
I'm certainly not the official developer of the project. But I will try to check the possibility of creating this function)

**Comment by CangShui** [comment]
> I'm certainly not the official developer of the project. But I will try to check the possibility of creating this function)

Excuse me, did you succeed?

**Comment by JadeVain** [comment]
> > I'm certainly not the official developer of the project. But I will try to check the possibility of creating this function)
> 
> Excuse me, did you succeed?

Not yet, the shotgun is spawn but can't be used

**Comment by chuxiaaaa** [comment]
BuyItemsServerRpc Can GetShotgun But Host Only

**Comment by Toaaa** [comment]
> BuyItemsServerRpc Can GetShotgun But Host Only
Then it's pretty useless.

**Comment by chuxiaaaa** [comment]
> > BuyItemsServerRpc Can GetShotgun But Host Only
> 
> Then it's pretty useless.

Yes, the host can directly generate shotguns without going through the store

**Comment by Toaaa** [comment]
> > > BuyItemsServerRpc Can GetShotgun But Host Only
> > 
> > 
> > Then it's pretty useless.
> 
> Yes, the host can directly generate shotguns without going through the store

You can simply give yourself credits and use them to buy the shotgun. Maybe the feature will be added in the future, maybe not. Personally, I think it's pretty useless. But if you want to implement it, go ahead.

**Comment by chuxiaaaa** [comment]
> > > > BuyItemsServerRpc Can GetShotgun But Host Only
> > > 
> > > 
> > > Then it's pretty useless.
> > 
> > 
> > Yes, the host can directly generate shotguns without going through the store
> 
> You can simply give yourself credits and use them to buy the shotgun. Maybe the feature will be added in the future, maybe not. Personally, I think it's pretty useless. But if you want to implement it, go ahead.

#91 The store function has been implemented here. If a shotgun is added to the game in the future, this function will automatically acquire the item.

**Comment by TheGreenBandit** [comment]
Seems to work spawning it with object spawner
